src/views/Project/index.jsx

```diff
@@ -36,6 +41,31 @@ const tagReposMapping = repositories =>
     },
   }),
 })
+@withStyles(theme => ({
+  root: {
+    background: theme.palette.background,
+  },
+  header: {
+    height: 60,
+    paddingBottom: theme.spacing.unit,
+  },
+  container: {
+    marginTop: 60,
+    paddingTop: theme.spacing.unit,
```

Comment:
Use `padding: 2 * theme.spacing.unit,` instead so the table and the expansion container are more breathable.